src/open_inwoner/js/components/accessibility/help_modal.js

```diff
@@ -11,16 +13,21 @@ class HelpModal {
     event.preventDefault()
     this.helpButton.classList.add('accessibility-header__modal--highlight')
     const modalId = document.getElementById('modal')
```

Comment:
See my previous question: if the `id` is so generic, how is this different from getting the modal via the class and `document.querySelector`?

Reply:
It will differentiate because this addClass will only happen on click on a very individual node. Click the selectors for each modal and you'll see how they look different, while retaining the same functionality.